.github/ISSUE_TEMPLATE/Request_new_generic_libraries_CAD_part.md

```diff
@@ -7,24 +7,30 @@ assignees: ''
 
 ---
 
-| |  |
+| Description | Field |
 |:---|:---|
 | Datasheet component  | [link or attachment ] |
 | project | [where we are going to assembly the part libraries] |
 | Alias Creo | [maker + code] |
 | IIT-Code | [if present] |
 | Applicant | name surname, @username |
 | supervisor | name surname, @username |
-| Assignees | name surname, @username |
 
-Attachment:
 
-Notes:
+### Attachment:
 
 
-### Madatory information/action:
-- [ ] Added assignees
-- [ ] Added label "Libraries-Request" (could be automatic )
-- [ ] Link/datasheet/drawing or other informations needing to create the libraries parts
+### Notes:
 
-without these informations is not possible proceed with the request issue
+
+### Checklist:
+- [ ] Created the Alias following [the standard convention](https://github.com/icub-tech-iit/cad-libraries/wiki/Mechanical-design-guidelines#commercial-components-coding-standard)
+- [ ] Added source files (STEP, IGS, etc)
```

Comment:
Coul be interesting merge the row below:
Added source files (STEP, IGS, etc or\and datasheet, drawing etc )